packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss

```diff
@@ -113,17 +126,10 @@
     @include add-color-icon($this-icon-object, $bgcolor);
     content: "";
     display: block;
-    // padding - optical spacing value
-    left: units($theme-site-margins-mobile-width) - units(
-        $theme-alert-bar-width
-      );
     position: absolute;
-    top: units($theme-alert-padding-y) * 0.75;
-    @include at-media($theme-site-margins-breakpoint) {
-      left: calc(
-        units($theme-site-margins-width) - units($theme-alert-bar-width)
-      );
-    }
+    // Adjusts the left position to accommodate the empty space built into the icon svgs
+    left: calc(units($theme-alert-icon-size) * -1 / 12);
```

Comment:
> [!Note]
> This adjustment was added because the alert icons have some empty padding built into the icon svgs. When the svg size is set to 24x24 pixels, the icon path is only 20x20 pixels. That means there are 2px of empty space on all sides of the svg, which means it needs a 1/12 (2px/24px) adjustment to allow the icon path to align with other content.

![image](https://github.com/user-attachments/assets/cb4c3eb0-e6eb-4b46-b235-d971df033451)
![image](https://github.com/user-attachments/assets/d089bcbf-9595-4c28-8c2a-37258ff12a9b)